### PR TITLE
Add event order related to insertFromPaste, fixes #117

### LINKED ISSUE #117: InputType to insert image from software keyboard
Android's software keyboard has a feature to insert image (https://developer.android.com/guide/topics/text/image-keyboard).

When using this, what is `InputType` for inserting image?  "`insertImage`"? 

Also, Windows 10 21H2 will seem to have same feature using Drag&Drop related API.

### PR DESCRIPTION
**Pull request by johanneswilm**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/input-events/pull/123.html" title="Last updated on Oct 8, 2021, 3:26 PM UTC (8753582)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/input-events/123/00c2127...8753582.html" title="Last updated on Oct 8, 2021, 3:26 PM UTC (8753582)">Diff</a>

### CODE REVIEW

**Comment by johanneswilm** [comment]
Hey @saschanaz you tend to be better than me concerning the correct lingo so I have included as a reviewer.

@travisleithead The issue here is that as far as I understand, the decision was that we wanted to say that there should be a `paste` event whenever we have a `insertFromPaste` `beforeinput` event. But we wanted to add this as a non-normative note so that it is just noted as a hint as we are not sure where the entire event order should be defined long-term.

File: index.html [hunk 1]
```diff
@@ -1336,6 +1336,15 @@ <h5>
           <div class="note">
             Other specifications may expand on this definition.
           </div>
+          <div class="note">
+            <p>
+              <i>This note is not normative.</i>
+            </p>
+            When an <code>"insertFromPaste"</code> <code>beforeinput</code>
```
Comment (saschanaz) [inline index.html]:
```suggestion
            When an <code>"insertFromPaste"</code> {{beforeinput}}
```

I think insertFromPaste should also be defined using `dfn` to allow linking, but out of scope here.

File: index.html [hunk 2]
```diff
@@ -1336,6 +1336,15 @@ <h5>
           <div class="note">
             Other specifications may expand on this definition.
           </div>
+          <div class="note">
+            <p>
+              <i>This note is not normative.</i>
```
Comment (saschanaz) [inline index.html]: Is this really non-normative?

File: index.html [hunk 3]
```diff
@@ -1336,6 +1336,15 @@ <h5>
...
+            </p>
+            When an <code>"insertFromPaste"</code> <code>beforeinput</code>
+            event is dispatched, it should be preceded by a
+            <a href="https://www.w3.org/TR/clipboard-apis/#clipboard-event-paste">paste</a>
```
Comment (saschanaz) [inline index.html]: Probably need to fix clipboard api spec to export `paste` definition.

File: index.html (hunk 2 above)
Comment (johanneswilm) [reply index.html]: I think you have a good point there @saschanaz . But the resolution was for it to be a non-normative note. Maybe we need to review that decision again, @travisleithead ?

Comment (johanneswilm) [reply index.html]: Agenda+

**Comment by johanneswilm** [comment]
Objective with PR has changed: https://www.w3.org/2021/09/24-editing-minutes.html#a03

I have checked all browsers [1] and the paste event is triggered before the beforeinput event on all four browsers.

[1] https://github.com/johanneswilm/input-events-testing

File: index.html (hunk 2 above)
Comment (johanneswilm) [reply index.html]: @saschanaz We have decided that it should be normative. The text has been changed accordingly.

File: index.html (hunk 3 above)
Comment (johanneswilm) [reply index.html]: @travisleithead @garykac Any change we can get the paste definition from the clipboard apis exported?

**Review by travisleithead — APPROVED** [review]
Awesome. Thanks for doing this.

**Comment by johanneswilm** [comment]
@travisleithead should we be migrating from travis to github actions? Or is there another policy in place?

**Pull request merged**